examples/fscloud/variables.tf

```diff
@@ -36,25 +32,18 @@ variable "bucket_hpcs_key_crn" {
 variable "resource_group" {
   type        = string
   description = "An existing resource group name to use for this example, if unset a new resource group will be created"
-  default     = null
 }
 
 variable "existing_at_instance_crn" {
   type        = string
   description = "Optionally pass an existing activity tracker instance CRN to use in the example. If not passed, a new instance will be provisioned"
-  default     = null
```

Comment:
leave `null` as default value because `existing_at_instance_crn` is not a required variable. If `null` is passed, the example will create a new activity tracker instance